azurelinuxagent/common/protocol/goal_state.py

```diff
@@ -22,11 +22,14 @@
 
 import azurelinuxagent.common.conf as conf
 import azurelinuxagent.common.logger as logger
-from azurelinuxagent.common.datacontract import set_properties, DataContract, DataContractList
+from azurelinuxagent.common.AgentGlobals import AgentGlobals
+from azurelinuxagent.common.datacontract import set_properties
+from azurelinuxagent.common.event import add_event, WALAEventOperation
+from azurelinuxagent.common.protocol.restapi import *
```

Comment:
This starred import was already there, I just used the `optimize import` option in PyCharm to sort it properly. Did you mean unpacking it and removing the starred import here?

Reply:
Yep, I mean avoiding `*` and importing only what's needed. But since you're saying you only changed the order of imports, I'm not pushing you to do it in this PR.

Reply:
There might be some files which import this file which inherently depend on the starred imports. They might be harder to find :/
I'll skip fixing this as part of this PR as that requires extra checks.